websites/A/Aodesu/presence.ts

```diff
@@ -0,0 +1,152 @@
+const presence = new Presence({
+		clientId: "1101840416866844672",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let iFrameVideo: boolean,
+	currentTime: number,
+	duration: number,
+	paused: boolean,
+	playback: boolean,
+	reproductor: string;
+
+enum Assets {
+	Logo = "https://i.imgur.com/CyVxys5.jpg",
+	Home = "https://i.imgur.com/i2EEGId.jpg",
+	Pause = "https://i.imgur.com/V8aOlhf.jpg",
+	Play = "https://i.imgur.com/vS09FhM.jpg",
+	Search = "https://i.imgur.com/BSHGBn1.jpg",
+	Emitting = "https://i.imgur.com/TNrq2vU.jpg",
+	Login = "https://i.imgur.com/aY4bjzh.jpg",
+	Register = "https://i.imgur.com/hdtAwWr.jpg",
+	Settings = "https://i.imgur.com/T91Uhm5.jpg",
+	Preview = "https://i.imgur.com/1RLd82M.jpg"
+}
+
+presence.on(
+	"iFrameData",
+	(data: {
+		iframeVideo: {
+			duration: number;
+			iFrameVideo: boolean;
+			currTime: number;
+			paused: boolean;
+			reproductor: string;
+		};
+	}) => {
+		playback = data.iframeVideo !== null;
+		reproductor = data.iframeVideo.reproductor;
+		if (playback) {
+			({ iFrameVideo, paused, duration } = data.iframeVideo);
+			currentTime = data.iframeVideo.currTime;
+		}
+	}
+);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: Assets.Logo,
+		startTimestamp: browsingTimestamp,
+	},
+	{ href, pathname } = window.location;
```

Comment:
This repository uses `document.location` rather than `window.location`
```suggestion
	{ href, pathname } = document.location;
```